src/utils/cart.js

```diff
@@ -0,0 +1,50 @@
+import builton from "./builton";
+import {getProductName} from "./productModifiers";
+
+const getCartSubProduct = (cartProduct, subProdId) => {
+  if(cartProduct && cartProduct._sub_products.length > 0 && subProdId) {
+    for (let i = 0; i < cartProduct._sub_products.length; i += 1) {
+      if (cartProduct._sub_products[i]._id.$oid === subProdId) {
+        return cartProduct._sub_products[i];
+      }
+    }
+  }
+
+  return undefined;
+};
+
+const getCartItems = () => {
+  const builtonCartItems = builton.cart.get();
+  const cartProducts = [];
```

Comment:
Could you add a comment on why we populate the `cartProducts` with the subproducts as well (basically explaining that we set the sizes as subproducts)